pyanaconda/packaging/yumpayload.py

```diff
@@ -1296,6 +1296,9 @@ def _applyYumSelections(self):
         else:
             self._selectYumGroup("core")
 
+        # Select the base group by default. It might be excluded later.
+        self._selectYumGroup("base")
```

Comment:
How does this work with the ``--nocore`` option? We don't want minimal (container) images to contain everything from the base group, do we?